### PR TITLE
SCB-1311 redesign database to make service faster and more ease of use

### PR DESCRIPTION
**Pull request by tianxiaoliang**

### CODE REVIEW

File: proxy.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```
Comment (WillemJiang) [inline proxy.sh]: License header.

File: pkg/model/mongodb_doc.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+type KVDoc struct {
+	ID        primitive.ObjectID `json:"_id,omitempty" bson:"_id,omitempty"`
+	LabelID   string             `json:"label_id,omitempty" bson:"label_id,omitempty"`
+	Key       string             `json:"key"`
+	Value     string             `json:"value,omitempty"`
+	ValueType string             `json:"value_type,omitempty" bson:"value_type,omitempty"` //ini,json,text,yaml,properties
+	Checker   string             `json:"check,omitempty"`                                  //python script
+
+	Labels   map[string]string `json:"labels,omitempty"` //redundant
+	Domain   string            `json:"domain,omitempty"` //redundant
```
Comment (WillemJiang) [inline pkg/model/mongodb_doc.go]: Can we treat the Domain string as a Label?

Comment (tianxiaoliang) [reply pkg/model/mongodb_doc.go]: it is tenant, we can not let user decide multi-tenant

**Comment by WillemJiang** [comment]
As we are using JIRA to track the tasks and features, please specify the JIRA number next time. 
I just created a [JIRA](https://issues.apache.org/jira/browse/SCB-1311) for this PR.

**Comment by commitcheck** [comment]
Hi @WillemJiang, I built an app that checks PR commits to ensure they have a JIRA number. It would prevent PRs like this being merged without JIRA numbers. It's free for OSS projects -> https://github.com/apps/commitcheck-app. Please let me know if you have any questions

**Pull request merged**